cli/src/pcluster/constants.py

```diff
@@ -111,6 +111,8 @@
 CW_DASHBOARD_ENABLED_DEFAULT = True
 CW_LOGS_ENABLED_DEFAULT = True
 CW_LOGS_RETENTION_DAYS_DEFAULT = 180
+CW_LOGS_ROTATION_ENABLED_DEFAULT = True
+CW_LOGS_RETENTION_DAYS_DEFAULT = 14
```

Comment:
I think this line needs to be removed?

Reply:
Good catch, it is something wrong I did when rebasing conflict, fixed